docs/resources/artifactory_remote_cargo_repository.md

```diff
@@ -13,6 +24,7 @@ for brevity sake, only cargo specific fields are included; for other fields see
 resource "artifactory_remote_cargo_repository" "my-remote-cargo" {
   key                 = "my-remote-cargo"
   anonymous_access    = true
+  url                 = "https://github.com/"
```

Comment:
I am not against these sorts of changes as part of this PR, but an explanation is in order

Reply:
it's a missing required field in the HCL example